### ISSUE TITLE
Multiple locations in resources

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#630]
Some resources have been published at multiple locations or even with multiple publishers with multiple locations like http://lobid.org/hbz01/HT019387889. At the moment it looks like this:
```
"publication" : [ {
    "location" : [ "New York ; London", "New York" ],
    "publishedBy" : [ "G. P. Putnam's Sons", "The Knickerbocker Press" ],
    "startDate" : "1915",
    "type" : [ "PublicationEvent" ]
  } ],
```
To separate the publishers it should look like this (like in [hbz01](http://lobid.org/hbz01/HT019387889)):
```
"publication" : [ {
    "location" : [ "New York ; London" ],
    "publishedBy" : [ "G. P. Putnam's Sons" ],
    "startDate" : "1915",
    "type" : [ "PublicationEvent" ]
 }, {
    "location" : [ "New York" ],
    "publishedBy" : [ "The Knickerbocker Press" ],
    "startDate" : "1915",
    "type" : [ "PublicationEvent" ]
 } ],
```
When this is finished the frontend should also be adjusted. At the moment the publication places and publishers aren't shown, see http://lobid.org/resources/HT019387889.

**Comment by ChristophEwertowski** [comment]
Note to myself: It has to work with a blank node approach although it lead to problems with the publication dates at the first try. Concating the MAB 419 fields and extracting the data doesn't work because note fields can originate from MAB 523. If it's a volume than MAB 425 should be taken, not MAB 419 (see [HT012996101 (MAB-XML)](http://lobid.org/hbz01/HT012996101) where the journal was founded 1998 but the volume is from 2000).
Since it's not urgent, I'm putting this issue into backlog.

**Comment by fsteeg** [comment]
> At the moment the publication places and publishers aren't shown, see http://lobid.org/resources/HT019387889.

Assigning myself for the UI part (note single values are shown, see: http://lobid.org/resources/HT016009930)

**Comment by fsteeg** [comment]
Deployed to stage: http://stage.lobid.org/resources/HT019387889

I think it's fine to handle this on the UI level only and leave the data as it is.

**Comment by ChristophEwertowski** [comment]
I also think that it's good enough. +1

[Also on the data level as you could see a printing of one edition of two publishers as one publication event.]

**Comment by fsteeg** [comment]
Resolved with 52415ab9ff1c6a10a974051be58fe6a759429736
